src/client/pages/NewAccountReview.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { MainLayout, buttonStyles } from '../layouts/Main';
+import { MainBodyText } from '../components/MainBodyText';
+import { Consent } from '@/shared/model/Consent';
+import { ToggleSwitchInput } from '../components/ToggleSwitchInput';
+import {
+	InformationBox,
+	InformationBoxText,
+} from '../components/InformationBox';
+import { ExternalLink } from '../components/ExternalLink';
+import locations from '@/shared/lib/locations';
+import { palette, space, textSans } from '@guardian/source-foundations';
+import { css } from '@emotion/react';
+import { Button } from '@guardian/source-react-components';
+import { buildUrlWithQueryParams } from '@/shared/lib/routeUtils';
+import { CmpConsentedStateHiddenInput } from '../components/CmpConsentStateHiddenInput';
+import { CsrfFormField } from '../components/CsrfFormField';
+import { QueryParams } from '@/shared/model/QueryParams';
```

Comment:
can these be updated to use the from `@/client/...` syntax?

same for other files modified here too